public/app/plugins/panel/heatmap/heatmap_tooltip.ts

```diff
@@ -83,7 +83,7 @@ export class HeatmapTooltip {
     let xData = data.buckets[xBucketIndex];
     // Search in special 'zero' bucket also
     let yData = _.find(xData.buckets, (bucket, bucketIndex) => {
-      return bucket.bounds.bottom === yBucketIndex || bucketIndex === yBucketIndex;
+      return bucket.bounds.bottom === yBucketIndex || bucketIndex === yBucketIndex.toString();
```

Comment:
why toString ?

Reply:
because `bucketIndex` is a string, but `yBucketIndex` is a Number.

Reply:
`bucketIndex` is an object key and when we iterate over it with lodash it returns it as string.